app/main.py

```
@@ -7,23 +7,16 @@
 
-# 3. 앱 시작 시 로깅 설정 적용
-from app.core.logging_config import setup_logging  # 추가
-setup_logging()  # 가장 먼저 호출
+# 앱 시작 시 로깅 설정 적용
+from app.core.logging_config import setup_logging
 
-# Settings를 초기화하여 .env 계층을 로드
+setup_logging()
+
+# Settings
 from app.core.settings import settings
```

Flake8 (7.3.0)

[error] 'app.core.settings.settings' imported but unused

(F401)